MAUI/DataGrid/Summaries.md

```diff
@@ -11,17 +11,647 @@ documentation: UG
+                                          SummaryType="DoubleAggregate" >
+                    <sfgrid:DataGridSummaryColumn.Template>
+                        <DataTemplate>
+                                    <StackLayout Orientation="Horizontal" BackgroundColor="Gray">
+                                        <Label Text="{Binding Converter={StaticResource SummaryConverter}, ConverterParameter = {x:Reference dataGrid} }" VerticalOptions="CenterAndExpand"
+                                    TextColor="White"
+                                   FontSize="Default"
+                                   VerticalTextAlignment="Center"
+                                   HorizontalTextAlignment="Start"
+                                   LineBreakMode="NoWrap"
+                                   HorizontalOptions="FillAndExpand">
+                                            <Label.Style>
+                                                <Style TargetType="Label">
+                                                    <Setter Property="FontAttributes" Value="Italic" />
+                                                </Style>
+                                            </Label.Style>
+                                        </Label>
+                                    </StackLayout>
+                                </DataTemplate>
+                        </sfgrid:DataGridSummaryColumn.Template>
+                    </sfgrid:DataGridSummaryColumn>
+                </sfgrid:DataGridSummaryRow.SummaryColumns>
+            </sfgrid:DataGridSummaryRow>
+        </sfgrid:SfDataGrid.CaptionSummaryRow> 
+    </sfgrid:SfDataGrid>
+
+{% endhighlight%}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# %}
+
+// To write a converter, follow the code example:
+
+
+public class GroupCaptionConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        var data = value != null ? value as Group : null;
+        if (data != null)
+        {
+            SfDataGrid dataGrid = (SfDataGrid)parameter;
+            var summaryText = SummaryCreator.GetSummaryDisplayTextForRow((value as Group).SummaryDetails, dataGrid.View);
+
+            return summaryText;
+        }
+        return null;
+    }
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return null;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![DataGrid with template loaded for individual caption summary column](Images\Caption-Summary\TemplateColumn.png)
+
+## Group summary
+
+Group summary values are calculated based on records in the group. The summary information will be displayed at the bottom of each group. You can view the group summary row by expanding the corresponding group header. The data grid adds any number of group summary row.
+
+Add group summary rows in the data grid by adding the [DataGridSummaryRow]() to [SfDataGrid.GroupSummaryRows]() collection.
```

Comment:
Group summary values are calculated based on the records within each group. The summary information will be displayed at the bottom of each group. You can view the group summary row by expanding the corresponding group header. The data grid allows for the addition of any number of group summary rows.

To add group summary rows in the data grid, you can add the [DataGridSummaryRow](https://chat.openai.com/) objects to the [SfDataGrid.GroupSummaryRows](https://chat.openai.com/) collection.

Reply:
Modified.